aries-site/src/examples/components/pageheader/PageHeaderIntroExample.js

```diff
@@ -1,6 +1,7 @@
 import React, { useContext } from 'react';
 import { Button, PageHeader, ResponsiveContext } from 'grommet';
-import { ActionMenu, ReverseAnchor } from '../../templates';
+import { ActionMenu } from 'aries-core';
+import { ReverseAnchor } from '../../templates';
```

Comment:
Should ReverseAnchor be in aries-core?

Reply:
Yes, for sure. There are a number of others as well, but just time-boxing and that didn't make it in yet.